client/src/store/Fetcher.ts

```diff
@@ -0,0 +1,5 @@
+import axios from 'axios';
+
+export const Fetcher = axios.create({
+  baseURL: 'http://localhost:4000/api/v1/',
```

Comment:
we should probably configure this, eventually...

Reply:
It's temporary until we'll have build setup.